### DISCUSSION TITLE
Why is this project called "OS" and not "panel"?

### DISCUSSION THREAD

**Discussion by MasterOfStar** [IceWhaleTech/CasaOS#602] [General]
It seems to me that a tool that provides management functions based on another OS cannot be called an "operating system". web panel would be a more accurate name, right?

**Comment by ClementNerma** [comment]
I guess it's because the goal of this project is to be your main environment - you don't interact much with the rest of the "true" OS anymore, you only use CasaOS. But I agree that it's a bit confusing.
